Firestore/core/include/firebase/firestore/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Hack to make the headers show up in IDEs
+# feature to make the headers show up in IDEs
```

Comment:
Ultranit: capitalize.

I'm mildly against this change, to be honest. If _"hack"_ is too strong a word, I'd suggest _"workaround"_. It's a genuine limitation of CMake.

Reply:
Let me fix it upstream. This is a transform to fix the native copybara leaker check. Essentially, we should not use the word in our code; but at least it will be replaced when doing the sync and thus kept internal.